### DISCUSSION TITLE
Get historical earnings-release date for a stock?

### DISCUSSION THREAD

**Discussion by Rapid1898-code** [dpguthrie/yahooquery#221] [General]
Hello - is it somehow possible to get the historical earnings release dates of a stock - eg. for AAPL?

I have this link - but there i can see only the occuring earnings releases per date:
https://finance.yahoo.com/calendar/earnings?from=2023-09-24&to=2023-09-30&day=2023-09-25

But i would need them for specific tickers?
Can this somewhere be found?

**Comment by ms82494** [comment]
Yes, and you were almost there. Just  need to enter your ticker in the input field halfway down the page. Here's a direct link:
https://finance.yahoo.com/calendar/earnings?symbol=AAPL
